ocs_ci/deployment/acm.py

```
@@ -188,8 +194,138 @@ def deploy_downstream(self):
+                    f"oc --kubeconfig {kubeconfig} rollout status "
+                    f"daemonset/submariner-routeagent "
+                    f"-n {constants.SUBMARINER_OPERATOR_NAMESPACE} --timeout=120s"
+                )
+
+                @retry(
+                    (CommandFailed, subprocess.TimeoutExpired),
+                    tries=3,
+                    delay=30,
+                    backoff=2,
+                )
+                def _run_rollout():
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_run_rollout`

Add return type annotation: `None`

(ANN202)